.github/actions/pack_streamlit_component_library/action.yaml

```diff
@@ -0,0 +1,54 @@
+name: 'Build Streamlit Component Library'
```

Comment:
This is not an unambiguous description in this context, because in the case of npm packages, building is the process of compiling Typescript code to Javascript, but we are packaging a package here.